runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -312,6 +313,36 @@ task validatesRunnerStreaming {
   ))
 }
 
+createCrossLanguageValidatesRunnerTask(
```

Comment:
This task makes all Dataflow runner v2 tests fail because the sdk container is cleaned up as soon as it is built. The cause is that in the common configuration, you have
config.startJobServer.finalizedBy config.cleanupJobServer
This will change the task dependency incorrectly.

Filed jira here: https://issues.apache.org/jira/browse/BEAM-11270

cc: @tysonjh